### PR TITLE
WinstonLutz commits for Elekta EPID compatibility

### PR DESCRIPTION
**Pull request by midamo**
I applied various fixes to make the WL analysis compatible with Elekta Agility and Iview EPID. Seems to work for me but I will continue validating the results later. Also, I didn't test my changes with other linacs... I may have broken something for someone else.

### CODE REVIEW

File: pylinac/winston_lutz.py [hunk 1]
```diff
@@ -366,6 +366,10 @@ def __init__(self, file):
         self.check_inversion()
         # Flip the image upside down... makes it more intuitive later on.
         self.flipud()
+        # The first pixel line on the left has a glitch on Elekta iview EPID
```
Comment (jrkerns) [inline pylinac/winston_lutz.py]: Do you have a sample dataset to show this effect? And how confident are you that this is true for all Elektas?

**Comment by jrkerns** [comment]
@midamo This is a good fix. Don't worry about testing for Varian; I have plenty of datasets there to make sure there are no bugs that break the system. Any further Elekta fixes are welcome, just upload the datasets that weren't working or that you based the fixes on to the [dropbox folder](https://www.dropbox.com/request/YKRu4AmuPsXu55uQq761) so I can add them to the test suite to make sure further changes don't break our progress =) Thanks!

File: pylinac/winston_lutz.py (hunk 1 above)
Comment (midamo) [inline pylinac/winston_lutz.py]: Need to double check with a few collegues... I should have a confirmation in a few days.

Comment (jrkerns) [inline pylinac/winston_lutz.py]: No problem. Once you confirm that and upload a dataset with the issue I'll merge.

Comment (midamo) [inline pylinac/winston_lutz.py]: The datasets already provided contains this glitch.

I am also able to confirm that this glitch is common for Elektas (4 hospitals, sample of ~ 15 linacs).   It doesn't appears on all of them though­. It affects the field_cax position by about 0.5 mm (for a 10x10 field). Clipping may not be only option, but it is a quick and safe one. An elekta EPID is 1024x1024 pixels.... 1023x1024 doesn't really affect the covered area. Feel free to use  another fix if you prefer.

Comment (jrkerns) [inline pylinac/winston_lutz.py]: Okay, that's fine. I just wanted to make sure this wasn't just your machine and no one else's. I'll merge this and add some tests and release a bugfix update. Thanks for your contribution!

**Pull request merged**